tests/test_lint.py

```diff
@@ -336,3 +336,21 @@ def test_bug_number_pings(content, num_nits):
     assert len(nits) == num_nits
     if num_nits > 0:
         assert set(["BUG_NUMBER"]) == set(v.check_name for v in nits)
+
+
+def test_redundant_pings():
+    """
+    Test that name contains '-ping' or 'ping-' or 'ping' or 'custom' yields lint errors.
+    """
+    content = {
+            "ping": {}
+    }
```

Comment:
The linter wants this reformatted.

```suggestion
    content = {"ping": {}}
```

(I really need to change CI to still run the tests regardless of lint failures)

Reply:
How to catch these errors myself, because `make lint` was successful locally

Reply:
```
❯ make lint
python3 -m flake8 glean_parser tests
python3 -m black --check glean_parser tests setup.py
would reformat tests/test_lint.py
Oh no! 💥 💔 💥
1 file would be reformatted, 30 files would be left unchanged.
make: *** [lint] Error 1
```

It does fail the lint test, but it's not perfectly obvious.
In #357 I did change it so that it shows the difference:

```
❯ make lint
python3 -m flake8 glean_parser tests
python3 -m black --check --diff glean_parser tests setup.py
--- tests/test_lint.py	2021-07-08 08:50:20.282340 +0000
+++ tests/test_lint.py	2021-07-08 08:50:49.756407 +0000
@@ -340,13 +340,11 @@

 def test_redundant_pings():
     """
     Test that name contains '-ping' or 'ping-' or 'ping' or 'custom' yields lint errors.
     """
-    content = {
-            "ping": {}
-    }
+    content = {"ping": {}}

     content = util.add_required_ping(content)
     all_pings = parser.parse_objects([content])
     errs = list(all_pings)
     assert len(errs) == 0
would reformat tests/test_lint.py
Oh no! 💥 💔 💥
1 file would be reformatted, 30 files would be left unchanged.
make: *** [lint] Error 1
```

That should now be more obvious.
You can even auto-format the files again:

```
❯ python3 -m black glean_parser tests setup.py
reformatted tests/test_lint.py
All done! ✨ 🍰 ✨
1 file reformatted, 30 files left unchanged.
```

We currently don't have a make task for that, but I should add it.

Reply:
Done: #358